### PR TITLE
Add global AWS data transfer resource and load usage from usage file

### PR DESCRIPTION
**Pull request by aliscott**

### CODE REVIEW

File: internal/config/environment.go [hunk 1]
```diff
@@ -29,6 +29,7 @@ type EnvironmentSpec struct {
 	TerraformVersion                    string   `json:"terraformVersion"`
 	TerraformRemoteExecutionModeEnabled bool     `json:"terraformRemoteExecutionModeEnabled"`
 	TerraformInfracostProviderEnabled   bool     `json:"terraformInfracostProviderEnabled"`
+	HasUsageFileResources               bool     `json:"hasUsageFileResources"`
```
Comment (alikhajeh1) [inline internal/config/environment.go:32]: just hasUsageFile?

**Review by alikhajeh1 — APPROVED** [review]
LGTM. Is this going to break things for users of the Infracost provider? If so, we should add deprecation notices and consider that for the version

**Comment by alikhajeh1** [comment]
Once this is merged, we need to create PRs to update the contributing guide, the docs, and let the contributors know via the community slack channel.

**Comment by aliscott** [comment]
> Is this going to break things for users of the Infracost provider?

This doesn't break the Infracost provider.

**Pull request merged**